src/test/java/kitchenpos/menu/acceptance/MenuAcceptanceTest.java

```diff
@@ -0,0 +1,110 @@
+package kitchenpos.menu.acceptance;
+
+import static kitchenpos.menu.acceptance.MenuGroupAcceptanceTest.메뉴그룹_등록_되어있음;
+import static kitchenpos.product.acceptance.ProductAcceptanceTest.상품_등록_되어있음;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import kitchenpos.AcceptanceTest;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+public class MenuAcceptanceTest extends AcceptanceTest {
+    private static final String URL = "/api/menus";
+
+    @Test
+    @DisplayName("메뉴를 관리한다.")
+    void manageMenu() {
+        // 상품 등록 되어 있음
+        Product 후라이드치킨 = 상품_등록_되어있음("후라이드치킨", BigDecimal.valueOf(10000));
```

Comment:
```suggestion
        Product 후라이드치킨 = 상품_등록_되어있음("후라이드치킨", 10000);
```

`상품_등록_되어있음` int -> BitDecimal 처리를 해주면 조금 더 편하게 메서드를 사용할 수 있을 것 같아요 ㅎㅎ